seatunnel-connectors-v2/connector-iceberg/pom.xml

```diff
@@ -120,6 +121,7 @@
         <dependency>
             <groupId>org.apache.flink</groupId>
             <artifactId>flink-shaded-hadoop-2</artifactId>
+            <scope>provided</scope>
```

Comment:
Add a note about this into iceberg connector docs?

Reply:
ok